### ISSUE TITLE
[Feature Request] run_hid_host BLE supported.

### ISSUE THREAD

**Issue by joyhope** [google/bumble#370]
run hid host current works for classic,  Is it feasible to support BLE mode? 

It is a good tool.

**Comment by barbibulle** [comment]
For BLE, the `examples/keyboard.py` includes a functional BLE keyboard host or device. There's no high level API for it, but the example is pretty simple and can make a good basis for an app or virtual HID. Adapting it to support a BLE mouse should also be pretty straightforward.

**Comment by joyhope** [comment]
Yes, I tried this keyboard, it is unable to connect to the iPhone.  I tried to understand why, so I needed some tools to find the reason. It is accepted by the iPhone as a keyboard, but unable to input anything.   I tried add mouse for this, but stopped in the keypad.

**Comment by barbibulle** [comment]
I have observed the same issue with the iPhone but didn’t have time to troubleshoot. It worked with Android for sure, and maybe with a Mac (not sure, it was a while back). For the iPhone, you might need to pair first.
